### PR TITLE
fix(integrations): Correct Admin permissions

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Fixes an issues causing project admins to be unable to configure the new slack integration.

### CODE REVIEW

**Review by dcramer — APPROVED** [review]

**Review by MaxBittker — APPROVED** [review]
lgtm but david should probably review - why is the change to integration_setup.py needed?

File: src/sentry/web/frontend/integration_setup.py [hunk 1]
```diff
@@ -3,15 +3,17 @@
 import logging
 
 from sentry.integrations.pipeline import IntegrationPipeline
-from sentry.web.frontend.base import BaseView
+from sentry.web.frontend.base import OrganizationView
```
Comment (dcramer) [inline src/sentry/web/frontend/integration_setup.py]: actually -- is organization slug even in the url path?

Comment (EvanPurkhiser) [reply src/sentry/web/frontend/integration_setup.py]: No actually. but afaict the OrganizationView pulls this from somewhere else. Again it's not strictly needed here, but it is consistent.

Comment (dcramer) [reply src/sentry/web/frontend/integration_setup.py]: it might pull from session when it can, but i think it might error if it cant

tl;dr integration_setup has no organization present in some cases, which is why this didnt use OrganizationView. I think you are going to have to somehow do this in the pipeline, since the pipeline is where it fetches the organization access from.

Comment (EvanPurkhiser) [reply src/sentry/web/frontend/integration_setup.py]: Nah this is fine, I'll just switch it back in that case. It didn't really need it, I kind of just wanted to this endpoint to be consistent. But yeah, you're right setup doesn't have the org.

Comment (EvanPurkhiser) [reply src/sentry/web/frontend/integration_setup.py]: 4b3fba1
